packages/app/resources/views/components/tenant-menu.blade.php

```diff
@@ -21,7 +21,10 @@
 
 {{ filament()->renderHook('tenant-menu.before') }}
 
-<x-filament::dropdown placement="bottom-start">
+<x-filament::dropdown
+    placement="bottom-start"
+    teleport
```

Comment:
Does the user menu also need teleporting? Also we should probably add a `filament-tenant-menu` class for consistency

Reply:
User menu doesn't need teleporting as it's not inside the sidebar.